operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -200,7 +278,27 @@ func (r *reconcilePVCExtensionRun) handleDelete() error {
 }
 
 func (r *reconcilePVCExtensionRun) handleCreate(claimName string, pvcConfig *platform.PersistentVolumeClaim) error {
-	size, err := parseResourceQuantityOr(pvcConfig.Size, defaultPVCSize)
+	// Before creating a PVC, verify if prerequisites are met. Currently there
+	// is only one requirement, a default storage class must exists or a
+	// storage class has to be specified explicitly. Since it's highly specific
+	// for PVCs only, it's implemented inside the extension, instead of
+	// collecting this information at the start and passing it into the
+	// extension.
+	//
+	// Note that to make this check less disruptive, in case if we face an
+	// error we still try to create a PVC.
+	hasDefault, err := utils.HasDefaultStorageClass(r.ctx, r.client)
+	if err == nil && !hasDefault && pvcConfig.StorageClassName == nil {
+		// For the backup PVC it's a hard stop
+		if r.target == PVCTargetCentralDBBackup {
+			r.log.Info("No default storage class or explicit storage class found, skip PVC creation")
```

Comment:
Not sure if this is the way to use this logger type, but please do something equivalent to:
```suggestion
			r.log.Infof("No default storage class or explicit storage class found, skipping %q PVC creation.", claimName)
```